demo/src/configs/sections/lovelace.ts

```diff
@@ -10,11 +10,12 @@ export const demoLovelaceSections: DemoConfig["lovelace"] = () => ({
       icon: "mdi:home-assistant",
       sections: [
         {
-          title: "Welcome 👋",
-          cards: [{ type: "custom:ha-demo-card" }],
-        },
-        {
+          // @ts-expect-error
           cards: [
+            document.location.search !== "?frontpage" && {
+              title: "Welcome 👋",
+              cards: [{ type: "custom:ha-demo-card" }],
+            },
```

Comment:
```suggestion
            ...document.location.search !== "?frontpage" ? [{
              title: "Welcome 👋",
              cards: [{ type: "custom:ha-demo-card" }],
            }] : [],
```
So we don't need `ts-ignore` and filter Boolean.